### ISSUE TITLE
Can't open libVTFLib13.so on Linux

### ISSUE THREAD

**Issue by RenaKunisaki** [REDxEYE/SourceIO#9]
When activating the addon:

```
Traceback (most recent call last):
  File "/usr/share/blender/2.80/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/home/rena/.config/blender/2.80/scripts/addons/SourceIO/__init__.py", line 9, in <module>
    from SourceIO.vtf.blender_material import BlenderMaterial
  File "/home/rena/.config/blender/2.80/scripts/addons/SourceIO/vtf/blender_material.py", line 5, in <module>
    from SourceIO.vtf.import_vtf import import_texture
  File "/home/rena/.config/blender/2.80/scripts/addons/SourceIO/vtf/import_vtf.py", line 6, in <module>
    from SourceIO.vtf.VTFWrapper import VTFLib
  File "/home/rena/.config/blender/2.80/scripts/addons/SourceIO/vtf/VTFWrapper/VTFLib.py", line 40, in <module>
    class VTFLib:
  File "/home/rena/.config/blender/2.80/scripts/addons/SourceIO/vtf/VTFWrapper/VTFLib.py", line 44, in VTFLib
    vtflib_cdll = cdll.LoadLibrary(vtf_lib_name)
  File "/usr/lib/python3.7/ctypes/__init__.py", line 434, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.7/ctypes/__init__.py", line 356, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libVTFLib13.so: cannot open shared object file: No such file or directory

```

Copying the file into `/usr/local/lib` doesn't help. What does fix it is changing `VTFLib.py` to use the full path to the file like it does on Windows:

```python
elif platform_name == "Linux":
    # On linux we assume this lib is in a predictable location
    # VTFLib Linux: https://github.com/panzi/VTFLib
    # requires: libtxc_dxtn
    full_path = os.path.dirname(__file__)
    vtf_lib_name = "libVTFLib13.so"

# ...

    elif platform.system() == "Linux":
        vtflib_cdll = cdll.LoadLibrary(os.path.join(full_path, vtf_lib_name))
```

**Comment by REDxEYE** [comment]
Done

**Comment by 0Curtain0** [comment]
**Blender 2.80 Official Release** 
**Pop!_OS** (Ubuntu Linux) **64bit**
**SourceIO** v**3.5.1** ( b86b029 - September 26th 2019 ) 

```
Traceback (most recent call last):
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/home/user/.config/blender/2.80/scripts/addons/SourceIO/__init__.py", line 15, in <module>
    from .vtf.blender_material import BlenderMaterial
  File "/home/user/.config/blender/2.80/scripts/addons/SourceIO/vtf/blender_material.py", line 5, in <module>
    from ..vtf.import_vtf import import_texture
  File "/home/user/.config/blender/2.80/scripts/addons/SourceIO/vtf/import_vtf.py", line 6, in <module>
    from ..vtf.VTFWrapper import VTFLib
  File "/home/user/.config/blender/2.80/scripts/addons/SourceIO/vtf/VTFWrapper/VTFLib.py", line 40, in <module>
    class VTFLib:
  File "/home/user/.config/blender/2.80/scripts/addons/SourceIO/vtf/VTFWrapper/VTFLib.py", line 44, in VTFLib
    vtflib_cdll = cdll.LoadLibrary(os.path.join(full_path, vtf_lib_name))
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/python/lib/python3.7/ctypes/__init__.py", line 434, in LoadLibrary
    return self._dlltype(name)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/python/lib/python3.7/ctypes/__init__.py", line 356, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/user/.config/blender/2.80/scripts/addons/SourceIO/vtf/VTFWrapper/libVTFLib13.so: cannot open shared object file: No such file or directory
```

**Comment by Ganonmaster** [comment]
@0Curtain0 

This is due to the Linux version of the VTFLib library not being included in the repository anymore. For some reason, @REDxEYE seems to have only included the Windows DLLs. This results in the addon not being able to load the library needed on Linux and thus preventing Blender from loading the VTF textures. I do not now why it was removed, but the solution would be to build the library yourself. 

The library can be found here: https://github.com/panzi/VTFLib  Make sure you have all the dependencies installed before you attempt to build. Once you have successfully built the library you should get a file named `libVTFLib13.so` and place it in [this directory](https://github.com/REDxEYE/SourceIO/tree/master/vtf/VTFWrapper).

As an aside: If there are people concerned with including untrusted binary versions of the library, we can setup automated builds for that repository through Github Actions, and automatically create reproducible binaries or OS packages. I would suggest the same for the Windows .dll files if possible.

**Comment by 0Curtain0** [comment]
@Ganonmaster 

> This is due to the Linux version of the VTFLib library not being included in the repository anymore. For some reason, @REDxEYE seems to have only included the Windows DLLs. This results in the addon not being able to load the library needed on Linux and thus preventing Blender from loading the VTF textures. I do not now why it was removed...

Hopefully it returns in the next release. 
For now, I will stick with the v3.5.1 release from July 30, 2019.
